Index: src/main/java/dcs/aber/ac/uk/csm2020_group_3/DatabaseHandler/RecordRemover.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package dcs.aber.ac.uk.csm2020_group_3.DatabaseHandler;\r\n\r\n/**\r\n * Removes student records from the db\r\n */\r\npublic class RecordRemover extends DatabaseHandler{\r\n\r\n    private String studentId;\r\n\r\n    private String studentPassword;\r\n\r\n    /**\r\n     *\r\n     * @param studentId id of the student being removed\r\n     * @param studentPassword student's password????\r\n     */\r\n    public RecordRemover(String studentId, String studentPassword) {\r\n        this.studentId = studentId;\r\n\r\n        this.studentPassword = studentPassword;\r\n\r\n    }\r\n\r\n    @Override\r\n    public void load() {;}\r\n\r\n    @Override\r\n    public void remove() {;}\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/dcs/aber/ac/uk/csm2020_group_3/DatabaseHandler/RecordRemover.java b/src/main/java/dcs/aber/ac/uk/csm2020_group_3/DatabaseHandler/RecordRemover.java
--- a/src/main/java/dcs/aber/ac/uk/csm2020_group_3/DatabaseHandler/RecordRemover.java	(revision a770cc09f555d29f65ad8bc1e893d1f15cd8b0e7)
+++ b/src/main/java/dcs/aber/ac/uk/csm2020_group_3/DatabaseHandler/RecordRemover.java	(date 1676541868359)
@@ -16,10 +16,10 @@
      */
     public RecordRemover(String studentId, String studentPassword) {
         this.studentId = studentId;
-
         this.studentPassword = studentPassword;
+    }
 
-    }
+
 
     @Override
     public void load() {;}
